integration_test/workflow_tests/test_packages/ingest/test_multuple_granules_happy_path.py

```diff
@@ -0,0 +1,197 @@
+import json
+import logging
+import time
+import uuid
+from unittest import TestCase
+
+import boto3
+
+import helpers
+
+
+class TestMultipleGranulesHappyPath(TestCase):
+    def test_multiple_granules_happy_path(self):
+        """
+        If multiple granules are provided, should store them in DB as well as in recovery bucket.
+        """
+        try:
+            # Set up Orca API resources
+            # ---
+            my_session = helpers.create_session()
+            granule_id_1 = uuid.uuid4().__str__()
+            granule_id_2 = uuid.uuid4().__str__()
+            provider_id = uuid.uuid4().__str__()
+            provider_name = uuid.uuid4().__str__()
+
+            collection_name = uuid.uuid4().__str__()
+            collection_version = uuid.uuid4().__str__()
+            bucket_name = "orca-sandbox-s3-provider"  # standard bucket where initial file exists
+            recovery_bucket_name = "rhrh-orca-primary"
+            excluded_filetype = []
+            key_name_1 = "PODAAC/SWOT/ancillary_data_input_forcing_ECCO_V4r4.tar.gz"
+            key_name_2 = "MOD09GQ/006/MOD09GQ.A2017025.h21v00.006.2017034065104.hdf"
+            execution_id = uuid.uuid4().__str__()
+
+            copy_to_archive_input = {
+                "payload": {
+                    "granules": [
+                    {
+                        "granuleId": granule_id_1,
+                        "createdAt": 628021800000,
```

Comment:
Avoid hardcoded test values here and elsewhere.

Reply:
changed